VideoLocker/AndroidManifest.xml

```diff
@@ -148,6 +148,7 @@
         <activity
             android:name=".view.WebViewFindCoursesActivity"
             android:label="@string/find_courses_title"
+            android:windowSoftInputMode="adjustPan"
```

Comment:
It shouldn't because of the way the webview is laid out in the layout. Also checked manually.